scripts/lib/common.sh

```diff
@@ -28,3 +28,68 @@ function display_timelines() {
     echo "TIMELINE: Conformance tests took $CONFORMANCE_DURATION seconds."
     echo "TIMELINE: Down processes took $DOWN_DURATION seconds."
 }
+
+function run_warm_ip_test() {
+    echo "Setting WARM_IP_TARGET=2 and MINUMUM_IP_TARGET=10"
+    echo "Running warm ip test"
+    $KUBECTL_PATH set env ds aws-node -n kube-system WARM_IP_TARGET=2
+    $KUBECTL_PATH set env ds aws-node -n kube-system MINIMUM_IP_TARGET=10
+    #Sleep a couple seconds to ensure propagation
+    sleep 2
```

Comment:
Pretty sure there is no delay for propogating the setting of an env var on the Daemonset resource... since `kubectl set env` simply reads the Daemonset's desired state and replaces the desired state with whatever was supplied, and writes the resource back to etcd...